pa11y/mappings/common/createCommonMappings.ts

```diff
@@ -0,0 +1,90 @@
+import { Options } from "wiremock-rest-client/dist/model/options.model";
+import wiremock from "../../lib/general";
+
+export default (baseUri?: string, options?: Options) => {
+  // Mapping for covers.
+  wiremock(baseUri, options).mappings.createMapping({
+    request: {
+      method: "GET",
+      urlPattern: "/api/v2/covers.*",
+    },
+    response: {
+      jsonBody: [
+        {
+          id: "870970-basis:134693959",
+          type: "pid",
+          imageUrls: {
+            small: {
+              url: "https://picsum.photos/98/160",
+              format: "jpeg",
+              size: "small",
+            },
+            large: {
+              url: "https://picsum.photos/142/172",
+              format: "jpeg",
+              size: "large",
+            },
+          },
+        },
+      ],
+    },
+  });
+
+  // Mapping for material list.
+  wiremock(baseUri, options).mappings.createMapping({
+    request: {
+      method: "HEAD",
+      urlPattern: "/list/default/.*",
+    },
+    response: {
+      jsonBody: {},
+    },
+  });
+
+  // Mapping for tokens (anonymous session).
+  wiremock(baseUri, options).mappings.createMapping({
+    request: {
+      method: "GET",
+      urlPath: "/dpl-react/user-tokens",
+    },
+    response: {
+      headers: {
+        "Content-Type": "application/txt",
```

Comment:
Please change the content type to `application/javascript`.

[This matches the actual response from the CMS](https://github.com/reload/dpl-cms/blob/opsaetning-af-a11y-check-i-gh-actions-pa-dpl-cms-DDFSOEG-324/web/modules/custom/dpl_react/src/Controller/DplReactController.php#L83-L83).

Reply:
👍